errorreporting/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I think we don't need this pom. This file must be removed!

Reply:
CC: @averikitsch to confirm

Reply:
Should I roll everything one directory up then?